src/main/java/com/firecontroller1847/truediscordlink/TrueDiscordLink.java

```diff
@@ -109,6 +109,29 @@ private void migrateConfigurations() throws IOException {
             migrated = true;
         }
 
+        // Migration for Channel Notification Upon Linkage.
+        if (!getConfig().contains("bot.linking.notify_linkage", true)) {
```

Comment:
For consistency, if there's any way we could put "this." in front of all "getConfig" calls, that'd be appreciated as well. My rule of thumb personally when coding is I always use "this." for instance methods, but I don't use "this." for instance members.